### PR TITLE
- adds missing navigation properties on composable functions in java

### PR DESCRIPTION
**Pull request by baywet**
## Summary

composable functions allow to navigate to the return type's navigation properties and so on.
This PR adds missing navigation properties on composable functions

## Generated code differences

https://github.com/microsoftgraph/msgraph-sdk-java/pull/466

## Command line arguments to run these changes

Provide the command line arguments here so that reviewers can quickly repro the results of changes.

## Links to issues or work items this PR addresses

### CODE REVIEW

File: test/Typewriter.Test/TestDataJava/com/microsoft/graph/requests/extensions/IEntityType3ForwardRequestBuilder.java [hunk 1]
```diff
@@ -28,4 +28,5 @@
      * @return the IEntityType3ForwardRequest instance
      */
     IEntityType3ForwardRequest buildRequest(final java.util.List<? extends com.microsoft.graph.options.Option> requestOptions);
+
```
Comment (zengin) [inline test/Typewriter.Test/TestDataJava/com/microsoft/graph/requests/extensions/IEntityType3ForwardRequestBuilder.java:31]: Can we add a composable function sample to the test data?

File: test/Typewriter.Test/TestDataJava/com/microsoft/graph/requests/extensions/IOnenotePageForwardRequestBuilder.java [hunk 2]
```diff
@@ -28,4 +28,5 @@
      * @return the IOnenotePageForwardRequest instance
      */
     IOnenotePageForwardRequest buildRequest(final java.util.List<? extends com.microsoft.graph.options.Option> requestOptions);
+
```
Comment (zengin) [inline test/Typewriter.Test/TestDataJava/com/microsoft/graph/requests/extensions/IOnenotePageForwardRequestBuilder.java:31]: T4 and whitespace management, I feel the pain :)

**Review by zengin — CHANGES_REQUESTED** [review]
Please consider improving the test data with composable functions as it seems that our current test metadata didn't have that.

File: test/Typewriter.Test/TestDataJava/com/microsoft/graph/requests/extensions/IOnenotePageForwardRequestBuilder.java (hunk 2 above)
Comment (baywet) [reply test/Typewriter.Test/TestDataJava/com/microsoft/graph/requests/extensions/IOnenotePageForwardRequestBuilder.java:31]: yep parsing the code is hard enough, keeping tabs on spaces (pun intended) is almost impossible.

File: test/Typewriter.Test/TestDataJava/com/microsoft/graph/requests/extensions/IEntityType3ForwardRequestBuilder.java (hunk 1 above)
Comment (baywet) [reply test/Typewriter.Test/TestDataJava/com/microsoft/graph/requests/extensions/IEntityType3ForwardRequestBuilder.java]: done!

**Comment by zengin** [comment]
Thanks for investing time to improve the test data, really appreciate it.

**Review by zengin — APPROVED** [review]

**Pull request merged**